connector-packager/connector_packager/xsd_validator.py

```diff
@@ -164,6 +163,21 @@ def validate_file_specific_rules(file_to_test: ConnectorFile, path_to_file: Path
                                                  " already exists. Cannot have multiple fields with the same name.")
 
                     return False
+                if field_name == 'instanceurl':
+                    used_for_oauth = False
+                    for conditions in child.iter("conditions"):
+                        for condition in conditions.iter("condition"):
+                            if 'field' in condition.attrib:
+                                if condition.attrib['field'] == 'authentication':
+                                    if 'value' in condition.attrib:
+                                        if condition.attrib['value'] == 'oauth':
+                                            used_for_oauth = True
+                    if not used_for_oauth:
+                        xml_violations_buffer.append("Element 'field', attribute 'name'='instanceurl' can only be used conditional on field " + 
+                                                     "'authentication' with 'value'='oauth'. See 'Connection Field Platform Integration' section " + 
+                                                     "of documentation for more information.")
+                        return False
+
                 field_names.add(field_name)
 
```

Comment:
Missing [changes](https://github.com/tableau/connector-plugin-sdk/commit/6bb847d3457bae5ccfdca29eb0cd95ffe75d787f).  Is current master merged into dev-2021.1?

Reply:
I see now there are merge conflicts listed below that need to be resolved, likely in dev-2021.1 first before this PR.

Reply:
I've submitted a new merge request to get changes from master to dev-2021.1 inorder to resolve conflict.

Reply:
https://github.com/tableau/connector-plugin-sdk/pull/757